src/components/PeopleFilters.tsx

```diff
@@ -1,12 +1,61 @@
-export const PeopleFilters = () => {
+/* eslint-disable no-restricted-syntax */
+import React from 'react';
+import classNames from 'classnames';
+import { Link, useSearchParams } from 'react-router-dom';
+import { Filter } from '../types/Filter';
+import { getSearchWith } from '../utils/searchHelper';
+
+export const PeopleFilters: React.FC = () => {
+  const [searchParams, setSearchParams] = useSearchParams();
+  const sex = searchParams.get(Filter.SEX) || null;
+  const name = searchParams.get(Filter.NAME) || '';
+  const centuries = searchParams.getAll('centuries') || null;
+
+  const centuriesValues = [16, 17, 18, 19, 20];
+
+  const resetParams = () => {
+    const newSearchParams = new URLSearchParams('');
+
+    return newSearchParams.toString();
+  };
+
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  function setParams(params: any) {
+    const search = getSearchWith(params, searchParams);
+
+    setSearchParams(search);
+  }
+
+  function handleQueryChange(e: React.ChangeEvent<HTMLInputElement>) {
+    setParams({ query: e.currentTarget.value || null });
+    searchParams.set(Filter.NAME, e.target.value);
+  }
+
   return (
     <nav className="panel">
       <p className="panel-heading">Filters</p>
 
       <p className="panel-tabs" data-cy="SexFilter">
-        <a className="is-active" href="#/people">All</a>
-        <a className="" href="#/people?sex=m">Male</a>
-        <a className="" href="#/people?sex=f">Female</a>
+        <Link
+          className={classNames({ 'is-active': sex === null })}
```

Comment:
why are you comparing with null? you can just check does sex param exist. If it doesn't you use `all` by default (or even don't use this param)